lib/sdr_client/redesigned_client/cli/credentials.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module SdrClient
+  class RedesignedClient
+    class CLI < Thor
+      # The stored credentials
+      class Credentials
```

Comment:
The credentials file is now only needed by the CLI since usage in applications will handle authorization via automagic token refreshes.